### PR TITLE
Add cognee config to telemetry

### PR DESCRIPTION
**Pull request by alekszievr**
image

## Summary by CodeRabbit

- **New Features**
	- Enhanced logging capabilities for pipeline execution, including detailed configuration data.
- **Bug Fixes**
	- Improved error handling and telemetry reporting during task execution.

### CODE REVIEW

File: cognee/modules/pipelines/operations/run_tasks.py [hunk 1]
```diff
@@ -190,7 +195,6 @@ async def run_tasks_with_telemetry(tasks: list[Task], data, pipeline_name: str):
 
 async def run_tasks(tasks: list[Task], data = None, pipeline_name: str = "default_pipeline"):
     config = get_current_settings()
```
Comment (borisarzentar) [inline cognee/modules/pipelines/operations/run_tasks.py:197]: Let's move also this config inside `run_tasks_with_telemetry`, then we don't have to pass it in.

File: cognee/modules/pipelines/operations/run_tasks.py [hunk 2]
```diff
@@ -159,15 +161,18 @@ async def run_tasks_base(tasks: list[Task], data = None, user: User = None):
...
     user = await get_default_user()
 
     try:
         logger.info("Pipeline run started: `%s`", pipeline_name)
         send_telemetry("Pipeline Run Started", user.id, {
             "pipeline_name": pipeline_name,
-        })
-
+        } | config
+        )
```
Comment (borisarzentar) [inline cognee/modules/pipelines/operations/run_tasks.py:174]: This formatting looks like there is some issue with the code. :) Maybe move the closing bracket `)` up with `config`.

**Review by dexters1 — APPROVED** [review]

File: cognee/modules/pipelines/operations/run_tasks.py (hunk 1 above)
Comment (alekszievr) [reply cognee/modules/pipelines/operations/run_tasks.py]: done

File: cognee/modules/pipelines/operations/run_tasks.py (hunk 2 above)
Comment (alekszievr) [reply cognee/modules/pipelines/operations/run_tasks.py]: done

**Review by Vasilije1990 — APPROVED** [review]

**Pull request merged**